src/resolvers/auth.js

```diff
@@ -1,15 +1,44 @@
 const bcrypt = require('bcryptjs');
 const models = require('../models');
+const { sendEmail } = require('../services/sendEmail');
 
 const {
+  authenticateResetPasswordToken,
   createNewAccessToken,
   createNewRefreshToken,
+  createNewPasswordResetToken,
   addAccessTokenCookie,
   addRefreshTokenCookie,
   clearAccessTokenCookie,
   clearRefreshTokenCookie,
 } = require('../middleware/auth');
 
+const { CLIENT_URL } = require('../config/config');
+
+const createResetPasswordEmail = (resetToken) => {
+  const resetPasswordUrl = `${CLIENT_URL}/resetPassword/${resetToken}`;
+  console.log(resetPasswordUrl)
+  return {
+    from: 'kevinzhang@uwblueprint.org',
+    subject: `Raising the Roof Password Reset Attempt`,
+    html: `Reset your password <a href="${resetPasswordUrl}"> here </a>`,
+  };
+};
+
+const createResetAttemptEmail = () => {
+  return {
+    from: 'kevinzhang@uwblueprint.org',
+    subject: `Raising the Roof Password Reset Attempt`,
+    html: `Someone recently attempted to reset your password at ${CLIENT_URL}, however, you do not have an account with us`,
```

Comment:
Not sure if this was a design choice but I'm wondering if we should have a call to action here? i.e. `Sign up for an account here` or something?

Reply:
@jzxhuang should I implement this?